.github/workflows/builds.yml

```diff
@@ -718,7 +718,7 @@ jobs:
                 zlib-devel
       # packaged Rust version has no profiler support built in, so get from rustup
       - name: Install Rust
-        run: curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.67.1 -y
+        run: curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.83 -y
```

Comment:
MSRV still tested other places, so ok

Reply:
yeah the point here was to make the coverage work. IIRC it needs the rust version to match the OS llvm version somehow. The older version failed, this works :man_shrugging:

Reply:
I wonder if it would mkse sense to use the system package here?  The Rust RPM appears to depend on llvm-libs, where the rustup version embeds it own. Might be a way to keep them in sync.  Or do that next time they are out of sync.